src/components/issue-description.component.js

```diff
@@ -1,6 +1,13 @@
 import React, { Component } from 'react';
-import { StyleSheet, View, ActivityIndicator } from 'react-native';
-import { ListItem } from 'react-native-elements';
+import {
+  Text,
+  StyleSheet,
+  View,
+  ActivityIndicator,
+  TouchableHighlight,
+} from 'react-native';
+import { ListItem, Button } from 'react-native-elements';
```

Comment:
I'm sorry, I was fixing merge conflicts here and included this `Button` as part of the conflict fix. Shouldn't be here since we're leveraging @machour's `<Button` component. My apologies 😞

Reply:
Np mate, thanks for the conflict fix 😄